confidence/confidence.py

```diff
@@ -177,18 +152,46 @@ def _evaluate(
             flag_metadata={"flag_key": flag_key},
         )
 
-    def _resolve(self, flag_name: FlagName, context: Dict[str, str]) -> ResolveResult:
+    def track(self, event_name: str, context: Dict[str, str]) -> None:
+        asyncio.create_task(self._send_event(event_name, context))
+
+    async def _send_event(self, event_name: str, context: Dict[str, str]) -> None:
+        current_time = datetime.utcnow().isoformat() + "Z"
+        request_body = {
+            "clientSecret": self._client_secret,
+            "sendTime": current_time,
+            "events": [
+                {
+                    "eventDefinition": f"eventDefinitions/{event_name}",
+                    "payload": {**self.context, **context},
+                    "eventTime": current_time,
+                }
+            ],
+            "sdk": {"id": "SDK_ID_PYTHON_CONFIDENCE", "version": __version__},
+        }
+
+        print(request_body)
+
+        event_url = "https://events.confidence.dev/v1/events:publish"
+        headers = {"Content-Type": "application/json", "Accept": "application/json"}
+        response = requests.post(event_url, json=request_body, headers=headers)
+        response.raise_for_status()
```

Comment:
where is this error handled?

Reply:
I think we need to figure out how to handle the response using some logging to signal if things are not being accepted.